benches/bench.rs

```diff
@@ -173,24 +173,34 @@ fn compare_serde(c: &mut Criterion) {
     group.bench_function("sr.capnproto.unpacked", |b| {
         b.iter(|| {
             black_box(&mut buffer).clear();
-            let mut message = ::capnp::message::Builder::new_default();
+
+            // Allocate just enough to fit the message. Note that we could optimize this even
+            // more by writing a custom implementation of `capnp::message::Allocator` that writes
+            // directly to `buffer`.
+            let allocator = ::capnp::message::HeapAllocator::new().first_segment_words(14);
```

Comment:
How do we get `14`? Feels very magical to me.

Reply:
It was determined from the size of the message.

However #5, then #6 make it unnecessary to use this constant.

Reply:
Ah, but even after those two patches it still seemed that capn proto is slower in deserialization time?

Reply:
That correct. My guess is that's mainly due to capnproto performing more validation of the input, and capnproto relying more on dynamic dispatch. I have some ideas about how to improve those things, but it will take some effort. (Part of the story is that to avoid dynamic dispatch, it looks like we really need Generic Associated Types. I've been playing around with that and considering a blog post to explain.)